src/lib/progress-spinner/progress-spinner.spec.ts

```diff
@@ -108,6 +108,29 @@ describe('MdProgressSpinner', () => {
     expect(progressElement.componentInstance.interdeterminateInterval).toBeFalsy();
   });
 
+  it('should allow a custom stoke width', () => {
+    let fixture = TestBed.createComponent(BasicProgressSpinner);
+    let pathElement = fixture.nativeElement.querySelector('path') as HTMLElement;
+
+    fixture.detectChanges();
+
+    expect(parseInt(pathElement.style.strokeWidth))
+      .toBe(10, 'Expected default stroke width to be 10px.');
+
+    fixture.componentInstance.strokeWidth = 20;
+    fixture.detectChanges();
+
+    expect(parseInt(pathElement.style.strokeWidth))
+      .toBe(20, 'Expected custom stroke width to be applied.');
+
+    fixture.componentInstance.strokeWidth = 0;
+    fixture.detectChanges();
+
+    expect(parseInt(pathElement.style.strokeWidth))
+      .toBe(10, 'Expected stroke width to fallback to 10px.');
```

Comment:
I'm not sure whether we should be testing the exact default value here since it'll start failing if we change it. I suggest either importing the `DEFAULT_STROKE_WIDTH` and checking against it, or just checking that it has a valid stroke width in the first place.